codespell_lib/__main__.py

```diff
@@ -6,4 +6,4 @@
     try:
         sys.exit(_script_main())
     except KeyboardInterrupt:
-        pass
+        sys.exit(f"\ncancelling '{sys.argv[0]}'\n")
```

Comment:
Is there a reason why `cancelling` isn't capitalized?

Reply:
Sure, that's fine to amend (I'll do this later, in case other changes are requested).